src/seedu/addressbook/commands/Command.java

```diff
@@ -55,6 +55,14 @@ public void setData(AddressBook addressBook, List<? extends ReadOnlyPerson> rele
     }
 
     /**
+     * Checks if method called is a mutating method
```

Comment:
Kudos for adding the header comments! This can be further simplified into "return true if...".  Also, as mentioned during the tutorial, it is better to enforce the overriding in all the subclasses. Sp think about how to enforce the overriding behaviour. Hint: adding one keyword to the method signature will work.